src/app/components/task/Task.js

```diff
@@ -206,7 +206,7 @@ class Task extends Component {
     const matchingTaskIndex = this.props.localResponses.findIndex(item => item.node.dbid === task.dbid);
     // deep copy the local responses object
     const mutatedLocalResponses = JSON.parse(JSON.stringify(this.props.localResponses));
-    if (task.type === 'single_choice') {
+    if (task.type === 'single_choice' && mutatedLocalResponses[matchingTaskIndex]?.node?.first_response_value) {
```

Comment:
Do we need to check for the presence of `mutatedLocalResponses[matchingTaskIndex]?.node?.first_response_value` or only `mutatedLocalResponses[matchingTaskIndex]?.node` is enough since we're assigning `first_response_value` below?